pmlc/.clang-format

```diff
@@ -1,2 +1,2 @@
 BasedOnStyle: LLVM
-AlwaysBreakTemplateDeclarations: Yes
+AlwaysBreakTemplateDeclarations: true
```

Comment:
Why is this needed? This seems to indicate you are using a different version of clang-format, we require version 9 at the moment.

Reply:
I added this because I can't get the formatting to run/commit checks to pass without this change. My clang-format version is 6 so yes that's probably the issue. However, I was under the impression the cenv should set the correct version, is this not the case? (Or, is it supposed to be the case and failing for me for some reason?)

Regardless I will update my clang-format and undo this.

Reply:
`clang-format` is not specified in our `environment.yml`. I haven't been able to find a suitable package for conda to manage it. So right now, we have to manually install it.